src/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -742,16 +745,19 @@ void activity_classifier::init_train(
                        {"features", flex_list(feature_column_names.begin(),
                                               feature_column_names.end())}});
 
+  bool use_data_augmentation = read_state<bool>("use_data_augmentation");
```

Comment:
Could we put this line after the comment, which applies to both this line and the actual call to `create_iterator`?